lib/12T/addf1/gf180mcu_osu_sc_12T_addf_1.lef

```diff
@@ -0,0 +1,172 @@
+# Copyright 2022 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+VERSION 5.7 ;
+BUSBITCHARS "[]" ;
+DIVIDERCHAR "/" ;
+
+MACRO gf180mcu_osu_sc_12T_addf_1
```

Comment:
This should be `<libraryname>__<cell name>` look at https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_sc_mcu7t5v0/blob/main/cells/addf/gf180mcu_fd_sc_mcu7t5v0__addf_2.lef#L15 which has `gf180mcu_fd_sc_mcu7t5v0__addf_2` see also the "cell name" section of https://bit.ly/open-source-pdks-naming